recipes/openpmix/meta.yaml

```diff
@@ -0,0 +1,131 @@
+{% set name = "openpmix" %}
+{% set version = "5.0.4" %}
+
+package:
+  # don't usually want top-level name to match any outputs,
+  # but staged-recipes doesn't resolve dependency properly without it;
+  # rename on feedstock
+  name: libpmix
+  version: {{ version }}
+
+source:
+  url: https://github.com/openpmix/openpmix/releases/download/v{{ version }}/pmix-{{ version }}.tar.bz2
+  sha256: f72d50a5ae9315751684ade8a8e9ac141ae5dd64a8652d594b9bee3531a91376
+
+build:
+  skip: true  # [win]
+  number: 0
+
+# one top-level build
+# outputs select which files to include
+# from this build
+# but don't run any build scripts themselves
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - make
+    - pkg-config
+    - python 3.12
+  host:
+    - libevent
+    - libhwloc
+    - zlib
+
+outputs:
+  - name: libpmix
+    files:
+      # libpmix has everything except for specific exclusions
+      include:
+        - '**/*'
+      exclude:
+        # these are in -bin
```

Comment:
While these kinds of comments may be helpful in development, they should be removed for the final `meta.yaml`.

Reply:
Since you mentioned documenting best practices, I'll provide a bit of background on why I think these (and other similar comments) should stay, and indeed be actively encouraged in best practices for recipes across conda-forge, in case it's helpful to your writing.

Most recipes are maintained in a tiny fraction of maintainers' time, and are often interacted with very infrequently, sometimes months or years between changes. Any deviation from the simplest recipes or bits that prompt "why is this here?" should have comments explaining what's going on. In this way, meta.yaml the same as any other code or config. Many times over the years, I've seen (and made!) what look like innocuous 'clean up' changes to recipes because something odd or unusual in meta.yaml was not explained, its rationale not remembered and not self-evident, and then removed or changed in a way which ultimately broke the recipe (examples include host dependencies that aren't linked, but required for build to succeed, parts of the recipe where changes must be kept in sync with another part of the recipe, etc.). Comments help explain these things and help maintainers keep recipes in good health, since it should not be expected that maintainers have all the history and context of the recipe in mind when it's time to make a contribution.

As a specific example, this particular comment helps maintainers and reviewers understand what should be packaged in which output vs excluded entirely, and what needs to be kept in sync. For example, if these exclusions are changed it is likely the `-bin` output inclusions are likely to need changes as well.